docs/units-and-global-variables.rst

```diff
@@ -310,3 +310,11 @@ available for a contract type ``C``:
     regular calls.
     The same restrictions as with ``.creationCode`` also apply for this
     property.
+
+Additionally to the properties that are available for contract types,
+a different set is available for interface type ``I``:
+
+``type(I).interfaceId``:
+    This property returns the ``EIP-165`` interface identifier of the given interface ``I``, excluding all
```

Comment:
Should link to https://eips.ethereum.org/EIPS/eip-165

Reply:
ah nice :)